src/api/app/views/webui2/webui/comment/_show.html.haml

```diff
@@ -1,5 +1,5 @@
 .comments
-  - commentable.comments.without_parent.each do |comment|
+  - commentable.comments.without_parent.includes(:user, :children).each do |comment|
```

Comment:
I think it would be even better to raise a query for every children but that loads user (It should be more efficient)